cluster-autoscaler/Dockerfile

```diff
@@ -15,6 +15,11 @@
 FROM gcr.io/google_containers/ubuntu-slim:0.1
 MAINTAINER Marcin Wielgus "mwielgus@google.com"
 
+RUN apt-get update -y && \
```

Comment:
Can we remove the ca cert we have in this [example](https://github.com/kubernetes/contrib/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#deployment-specification) since we no longer need it.